docs/fast_data/configuration/realtime_updater.md

```diff
@@ -239,6 +253,15 @@ The ER Schema ConfigMap is created after the service is attached to a System of
 This is an empty configuration: the Real-Time Updater Microservice could be deployed without pod restart, but this file must be modified according to the projections associated with this microservice to work properly.
 :::  
 
+:::tip Shared ER Schema
+
+If you have already [attached an ER Schema to a Single View Creator](/fast_data/configuration/single_view_creator.md#selecting-an-er-schema-with-the-no-code), you can re-use it in the Real-Time Updater configuration too.
+
+By deleting the default one and [add an existing Config Map](/development_suite/api-console/api-design/services.md#shared-configmaps) corresponding to the one of the Er Schema, changes made to an Er Schema will be reflected also in the Real-Time Updater. 
```

Comment:
If possible, I would also state that it is not recommended to change the ER schema from micro-services section. Instead users should rather exploit the dedicated section in Fast Data configurator.

Reply:
Sure, I can put a warning panel

Reply:
@danibix95 and @TheRusCompass any other comment on that?
We want to merge this PR asap (if possible)